### ISSUE TITLE
Cross-publish for javascript

### ISSUE THREAD

**Issue by ryan-williams** [lasersonlab/ndarray.scala#12]
All dependencies of [the `zarr` module](https://github.com/lasersonlab/ndarray.scala/tree/master/zarr) are cross-published for JVM and JS except [JBlosc](https://github.com/lasersonlab/jblosc/).

Cross-publishing to JS would be "nice to have", either for use on the client in #11, and also to prepare for adding a scala-native target later.

Native support is currently blocked on [Cats](https://github.com/typelevel/cats/issues/1549), which in turn [needs ScalaCheck](https://github.com/rickynils/scalacheck/issues/396), and so is probably a few months out at least, but it would be really cool as a potential way to support R as well, or even be usable from Python, in case features make it in here that aren't in the reference python implementation.
